test/inspector/inspector-helper.js

```diff
@@ -393,7 +393,7 @@ Harness.prototype.addStderrFilter = function(regexp, callback) {
 
 Harness.prototype.assertStillAlive = function() {
   assert.strictEqual(this.running_, true,
-                     'Child died: ' + JSON.stringify(this.result_));
+                     `Child died: ${JSON.stringify(this.result_)}`);
```

Comment:
Lexically this has reasonable affinity with #13684, so makes sense to change the wordings, though through another PR - as the objective of this one is to introduce template strings and is met.